sql/2022/performance/gaming_metric.sql

```diff
@@ -20,6 +20,6 @@ SELECT
   SUM(IF(JSON_EXTRACT_SCALAR(payload, '$._performance.gaming_metrics.lcpOverlaySoft') = 'true', 1, 0)) AS lcp_overlay_soft,
   SUM(IF(JSON_EXTRACT_SCALAR(payload, '$._performance.gaming_metrics.lcpOverlaySoft') = 'true', 1, 0)) / COUNT(0) AS lcp_overlay_soft_per
 FROM
-  `httparchive.pages.2022_07_01_*`
+  `httparchive.pages.2022_07_01_*` -- noqa: L062
```

Comment:
@rviscomi @konfirmed should this be using 2022_07_01 or 2022_06_01?

Reply:
Should be 06

Reply:
Updated and rerun. though [some of your comments here](https://docs.google.com/spreadsheets/d/1TPA_4xRTBB2fQZaBPZHVFvD0ikrR-4sNkfJfUEpjibs/edit#gid=1079156379) are now slightly off and I can't edit them.

Reply:
Thanks! Updated the comments